### PR TITLE
[Feature] Add run from docker

### PR DESCRIPTION
**Pull request by JayJamieson**
# Overview
You can build a basic docker image and installing all dependencies in container with

```bash
docker build -t bleep .

# tag the image ready for push
docker tag bleep autobleep/bleep:latest

# push image to registry
docker push autobleep/bleep:latest
```

Run docker image with input and output mounts to host machine

```bash
docker run --rm -it --volume "$(pwd)/examples/example1.mkv:/input/input.mkv" --volume "$(pwd)/output:/output" bleep
```

## Known Issue

I haven't done much with local mounting and output if internally generated files and permission carry over. Currently this build command through docker will output a file with `root:root` permission set. You can fix this by `chown` the folder and file.

I was trying to get https://mydeveloperplanet.com/2022/10/19/docker-files-and-volumes-permission-denied/ to work but was unable to get it functioning with time I had.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -17,7 +17,6 @@ matplotlib==3.7.0
 more-itertools==9.1.0
 mypy-extensions==1.0.0
 numpy==1.24.2
-openai-whisper @ git+https://github.com/openai/whisper.git@7858aa9c08d98f75575035ecd6481f462d66ca27
```
Comment (cmgriffing) [inline requirements.txt]: I'm curious why this was removed. Could have been a mistake on my part, but just want to make sure.

Comment (JayJamieson) [reply requirements.txt]: I had an issue where installing `whisper-timestamped` didn't like the specific static version of `openai-whisper`. Removed it and it seemed to infer what it needed and installed perfectly after removing. 

I could very well be in the wrong here. Try the build and let me know if it's needed.

File: autobleep/autobleep.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ class AutoBleep:
     def __init__(
         self,
         input,
-        output="./output/output.mka",
+        output,
```
Comment (cmgriffing) [inline autobleep/autobleep.py]: Is there a reason we needed to remove the default?

File: requirements.txt (hunk 1 above)
Comment (cmgriffing) [reply requirements.txt]: I think this got fixed in a separate PR. Maybe pull in the latest and fix merge conflicts?

Comment (JayJamieson) [reply requirements.txt]: perfect, I can manage that.

File: autobleep/autobleep.py (hunk 2 above)
Comment (JayJamieson) [reply autobleep/autobleep.py]: Good spotting. I'll get it fixed, it wasn't needed for when running it through docker but thats not the cases if you use python only.

**Pull request merged**